Documentation/install/upgrade.rst

```diff
@@ -41,7 +41,7 @@ file.
         --namespace=kube-system \\
         --set preflight.enabled=true \\
         --set agent.enabled=false \\
-        --set config.enabled=false \\
+        --set enabled=false \\
```

Comment:
Correct, this is the equivalent to:
```yaml
agent:
  enabled:
```

All of the config for 'agent' is now a top-level value. This will be well-documented in the upgrade guide which I will complete this week, along with examples using helm --set and YAML.

Reply:
`--set enabled=false` is incredibly non-intuitive, let's call it at least `enableAgent` or something along those lines.

Reply:
I'm actually confused now, the PR seems to replace `config.enabled` with `enabled` but the only usage of `.Values.enabled` is in `cilium-agent-clusterrole.yaml`.

Reply:
the flag has been renamed and properly used in the relative yaml files